packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -182,10 +182,10 @@ class SemanticsData with Diagnosticable {
   /// The [flags], [actions], [label], and [Rect] arguments must not be null.
   ///
   /// If [label] is not empty, then [textDirection] must also not be null.
-  const SemanticsData({
+  SemanticsData({
```

Comment:
Why can this be no longer const?

Reply:
Oh, because the asserts are accessing `attributedLabel.string`? I wonder if we could use attributedLabel being non-null as a signal that you need a text direction?

Reply:
We can make this const if we:
1. makes AttributedString to have a const constructor 
2. makes all the attributedLabel/Value/Hint to be nullable.

If we do (1), we lose ability to verify whether the AttributedString is valid during the constructor, and will make handling AttributedString a bit harder.

If we do (2), the label/value/hint needs will be non null after SemanticsNode, make it nullable will probably break more APIs, for example the SemanticsNode.label will be String?